public/schemas/capital.json.erb

```diff
@@ -0,0 +1,61 @@
+{
+  "id": "file://#{@schema_dir}/capital.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Legal Aid Check Financial Eligibility add capital details to an assessment schema",
+  "type": "object",
+  "properties": {
+    "bank_accounts": {
+      "description": "Describes the name of the bank account and the lowest balance during the computation period",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "description": {
+            "description": "The name of the bank account",
+            "type": "string"
+          },
+          "value": {
+            "description": "The lowest balance on the account during the caluclation period",
+            "neg_or_pos_number_with_two_decimals": {
+              "description": "A negative or positive number (including zero) with two decimal places",
+              "type": "number",
+              "multipleOf": 0.01
```

Comment:
Apologies @chris-groves , I don't think this will work. I think the tests are failing because the tests are generating values in the format 0.44e5 (and this is meaning the regex pattern match fails). I think apply is also sending values in this format so I don’t think we can use the regex and will have to go back to using multipleOf - sorry!